server/api/mailgun/handler.js

```diff
@@ -62,30 +62,41 @@ module.exports.send = (person, event, type) => new Promise((resolve, reject) =>
 					`Looking forward to seeing you!\n` +
 					`Stanford Raagapella`,
 
-			html: `<h3>Hey ${person["First Name"]}!</h3>` +
+			html: `<h3>Hi ${person["First Name"]}! </h3>` +
```

Comment:
Extra space here but IDC Ill merge it anyway -- can't do much from my phone abroad

Reply:
Ah- I added that on purpose because of how gmail interprets and previews